src/main/groovy/io/seqera/wave/controller/ViewController.groovy

```diff
@@ -75,6 +75,8 @@ class ViewController {
         final binding = new HashMap(20)
         binding.build_id = result.buildId
         binding.build_success = result.succeeded()
+        binding.build_failed = result.exitStatus  && result.exitStatus != 0
+        binding.build_in_progress = result.exitStatus == null
         binding.build_exit_status = result.exitStatus
```

Comment:
Considering we are going to add also a [running](https://github.com/seqeralabs/wave/issues/603) status, it could make sense to encapsulate this logic into `WaveBuildRecord` adding a `Status getStatus()` method

Reply:
done